pom.xml

```diff
@@ -6,7 +6,7 @@
   <modelVersion>4.0.0</modelVersion>
 
    <groupId>ome</groupId>
-   <version>5.2.0</version>
```

Comment:
Understodd. I think the `5.2.0` -> `5.2.4` is confusing even though it matches the underlying library version bump. Could you bump the `bio-formats-jace` version from `5.2.0` to `5.2.1` instead? Also, do we need to upload artifacts anywhere?